_locales/_untranslated_en.json

```
@@ -32,6 +32,12 @@
   "pref_donate": {
     "message": "Donate"
   },
+  "pref_disable_idle_title": {
+    "message": "Disable IMAP IDLE"
+  },
+  "pref_disable_idle_explain": {
+    "message": "Do not use IMAP IDLE extension even if the server supports it. Enabling this option will delay message retrieval, enable it only for testing."
```

there may be some confusion wrt enable/disable and it/this. here my attempt to clarify wording:

```suggestion
    "message": "Skip IMAP IDLE"
  },
  "pref_disable_idle_explain": {
    "message": "Use polling even if the server supports IMAP IDLE. This will delay message retrieval. Use for testing only."
```